mlflow/pyfunc/__init__.py

```diff
@@ -372,6 +372,19 @@ def predict(self, data: PyFuncInput) -> PyFuncOutput:
             data = _enforce_schema(data, input_schema)
         return self._predict_fn(data)
 
+    def unwrap_python_model(self, unwrap_metadata: bool = False):
+        """
+        Unwrap the underlying Python model object, and it's metadata (optionally).
+
+        :param unwrap_metadata: (optional) If True, return a tuple of the
+        underlying Python model object and metadata loaded from the MLmodel file.
+        """
+        return (
+            self._model_impl.python_model
```

Comment:
Let's wrap this in a try / catch where we raise an exception if the value returned from this is None and pass the underlying exception along with the message.